packages/sdk/src/models/market.ts

```diff
@@ -53,10 +53,11 @@ class Market {
   public scalarType: ScalarRangeType | null;
   /** The resolved outcome for the market. */
   public resolvedOutcome: OutcomeReport | null;
-
+  /** IPFS cid for market metadata */
+  public metadata: string;
```

Comment:
Can this be renamed to something more descriptive e.g. `metadataCid` since we use `metadata` in the code to refer to `DecodedMarketMetadata` object and similar?

Reply:
They're actually same. Just that `img` when comes with other market data slows down overall fetching.